app/src/organisms/Devices/PipetteCard/AboutPipetteSlideout.tsx

```diff
@@ -58,13 +58,15 @@ export const AboutPipetteSlideout = (
               as="h6"
               fontWeight={TYPOGRAPHY.fontWeightSemiBold}
               color={COLORS.darkGreyEnabled}
+              textTransform={TYPOGRAPHY.textTransformUppercase}
             >
               {t('current_version')}
             </StyledText>
             <StyledText
               as="p"
               paddingTop={SPACING.spacing4}
-              paddingBottom={SPACING.spacing12}
+              paddingBottom={SPACING.spacing16}
+              color={COLORS.darkBlackEnabled}
```

Comment:
```suggestion
              color={COLORS.darkBlackEnabled}
```

text's default color is `COLORS.darkBlackEnabled` so we don't need to add this line.

https://github.com/Opentrons/opentrons/blob/edge/app/src/atoms/GlobalStyle/index.ts#L28

Reply:
I'm actually seeing that the text is rendering in darkGreyEnabled if nothing is set explicitly for the `color` prop. Is it ok to leave it in?

Reply:
yeah, it is okay

Reply:
nevermind! fixed— the parent Flex box was set to darkGreyEnabled